src/sidebar/oauth-auth.js

```diff
@@ -37,6 +37,7 @@ function auth($http, settings) {
     return {
       accessToken:  data.access_token,
       expiresIn:    data.expires_in,
+      expiresAt:    new Date(Date.now() + (data.expires_in * 1000)),
```

Comment:
The easiest way to avoid duplicating this code for computing `expiresAt` (better than moving it into a function and then having to call that function from multiple places) was to add it to the `tokenInfo` object.